float32 delta_t
float32 avg_delta_t
float32 avg_freq
uint64 iterations